tests/mark/check_regression_decorator.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
Is this just a copy of the parametrization tests? Doesn't seem to add any checks for the `@regression` decorator.